GOVERNANCE.md

```diff
@@ -0,0 +1,80 @@
+
+#### Register to vote
+Participants may register to vote on changes to the AIGF Standard (see [Section 2](#2decision-making) below). To do so register to the group by sending an empty email to [ai-governance-framework+subscribe@lists.finos.org] to join the enrolled voting participants group.
+
+Upon enrollment as an [AIGF voting participant](https://github.com/orgs/finos/teams/ai-governance-framework-participants), you will be invited to join the [FINOS GitHub organization](https://github.com/orgs/finos/people) and the [aigf-participants](https://github.com/orgs/finos/teams/ai-governance-framework-participants) GitHub team.
+
+- [View the AIGF Participants GitHub Team](https://github.com/orgs/finos/teams/ai-governance-framework-participants)
+
+### 1.2. Discussion Groups
+
+The Working Group may form one or more "Discussion Groups" to organize collaboration around a particular aspect of a specification. Discussion Groups are for discussion only.  Approval of all portions of a specification is subject to the consensus-based decision-making process of the AIGF Standard Working Group.
+
+### 1.3. Maintainers & Editors
+
+“Maintainers” are responsible for organizing activities around developing, maintaining, and updating the specification(s) developed by the Working Group.  Maintainers are also responsible for determining consensus and coordinating appeals.  The Working Group has designates six Maintainer(s). The Working Group may select a new or additional Maintainer(s) upon Approval of the Working Group Participants.  
+
+- [View the AIGF Maintainers GitHub Team](https://github.com/orgs/finos/teams/ai-governance-framework-maintainers)
+
+“Editors” are responsible for ensuring that the contents of the document accurately reflect the decisions that have been made by the group, and that the specification adheres to formatting and content guidelines. The Working Group will designate one or more Editor(s). The Working Group may select a new Editor upon Approval of the Working Group Participants.
+
+- [View the AIGF Editors GitHub Team](https://github.com/orgs/finos/teams/ai-governance-framework-editors)
+
+#### How do you become an Editor or Maintainer?
+
+Once you are an enrolled participant in AIGF, you can apply to become an **editor** or **maintainer** by contacting the existing AIGF maintainers at [ai-governance-framework-maintainers@finos.org](mailto:ai-governance-framework-maintainers@finos.org) and then seeking the approval of the AIGF Standards Working Group. Generally, the maintainers will look for both a history of contribution to AIGF and a commitment to investing sufficient time in the role from any prospective candidates before proposing them to the Standards Working Group for approval. 
```

Comment:
The ai-governance-framework-maintainers@finos.org email distribution list doesn't work @lucaborella89 can you please create this?